FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -117,17 +128,16 @@ static string QueryString(IReadOnlyDictionary<string, string> query)
             return string.Join("&", keyValues);
         }
 
-        static HttpClient CreateClient(Uri endpoint, TimeSpan timeout)
+        static HttpClient CreateClient(Uri endpoint, TimeSpan timeout, HttpClient httpClient)
```

Comment:
remove `HttpClient httpClient`

Reply:
have to pass it from FaunaClient() constructor

Reply:
and this can change as a result of the last one